src/mlpack/core/data/binarize.hpp

```diff
@@ -47,10 +47,10 @@ void Binarize(const arma::Mat<T>& input,
   #pragma omp parallel for
   for (size_t i = 0; i < totalElems; ++i)
```

Comment:
please change size_t to int, because openMP of vc do not support unsigned int yet